salt/metalk8s/orchestrate/etcd.sls

```diff
@@ -32,6 +32,7 @@ Check etcd cluster health for {{ node }}:
     - metalk8s_etcd.check_etcd_health:
       - minion_id: {{ node }}
     - attemps: 5
+    - until: "cluster is healthy"
```

Comment:
Where does this come from?

Reply:
Was from a TEST commit where I was trying stuff to make the expansion working.

The idea was to execute `state.running` to check if something was running after restarting the minion (if not == then startup is over, we can proceed with the other states).

To do so, I had to add the support for `until` to `metalk8s.module_run` and thus update its use here.

But in the end it didn't work, so it was removed ;)

Now trying to use a dedicated parallel state for startup (seems to be working so far).